java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -383,9 +391,7 @@ abstract class AbstractArcHost(vararg initialParticles: ParticleRegistration) :
      * Unregisters [Handle]s from [StorageProxy]s, and clears references to them from [Particle]s.
      */
     private suspend fun cleanupHandles(handles: HandleHolder) {
-        for ((name, handle) in handles.handles) {
-            // TODO: disconnect/unregister handle
-        }
+        // TODO: disconnect/unregister handles
```

Comment:
This reminds me, we need to file an issue, as we need an api to tell handles to unregister themselves, and some way to tell the storageProxy/ActiveStore it is no longer needed and can close its resources.

Reply:
I am working on handle start/stop API now, just created a ticket for it. 

I'll create a ticket for the storageProxy / store work as well.